app/src/main/java/org/covidwatch/android/ui/home/HomeFragment.kt

```diff
@@ -30,7 +33,19 @@ class HomeFragment : BaseFragment<FragmentHomeBinding>() {
         homeViewModel.navigateToOnboardingEvent.observe(viewLifecycleOwner, EventObserver {
             findNavController().navigate(R.id.splashFragment)
         })
-        homeViewModel.exposureSummary.observe(viewLifecycleOwner, Observer(::bindExposureSummary))
```

Comment:
Observing `exposureSummary` gives us updates when you change the value. There is no need to remove it and getting direct value from `sharedPreferences.exposureSummary` as it's the same value as in `homeViewModel.exposureSummary`. 

Return how it used to be please.

Reply:
I did not remove the exposureSummary observer -- it is in the else clause.  From what you are saying I should remove the "else".  The reason for getting the exposure summary from preferences is that when I didn't do that and left the updating of the exposure statistics screen to the observer, it didn't happen -- or at least not right away.  This also explains my use of the "retrieved" boolean - I want the statistics to be obtained directly from  Preferences only when we have just gotten random data.

Reply:
So I restored the homeViewModel.exposureSummary.observe statement without the "else" and it basically set the exposureSummary values to what they were before I set the preferences to the random values.  I tried placing the observer statement before I set the preferences, and the same thing happened. So I'm mystified. Any ideas?